img/loading-squares.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 150 150"><style>.c{transform-origin:50px 50px;transform:rotate(0);animation:c 4s infinite}@keyframes c{0%{transform:rotate(0)}90%{transform:rotate(0)}100%{transform:rotate(90deg)}}rect{width:30px;height:30px;opacity:.3;transform-origin:var(--o);fill:currentColor;animation:b 4s infinite var(--d);}@keyframes b{0%{transform:rotate(0);opacity:.3}5%{opacity:1}10%,100% {transform:rotate(90deg);opacity:.3}}</style><g transform="translate(25 25) rotate(45 50 50)"><g class="c"><rect x="0" y="0" style="--o:15px 15px;--d:0s"/><rect x="35" y="0" style="--o:50px 15px;--d:.4s"/><rect x="70" y="0" style="--o:85px 15px;--d:.8s"/><rect x="0" y="35" style="--o:15px 50px;--d:1.2s "/><rect x="35" y="35" style="--o:50px 50px;--d:1.6s "/><rect x="70" y="35" style="--o:85px 50px;--d:2s "/><rect x="0" y="70" style="--o:15px 85px;--d:2.4s "/><rect x="35" y="70" style="--o:50px 85px;--d:2.8s "/><rect x="70" y="70" style="--o:85px 85px;--d:3.2s "/></g></g></svg>
```

Comment:
I think you could add a `svg{color:#???}` declaration in your `<style>` element in order to "show" the `currentColor` usage.

Reply:
Done